ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_general.schema.yml

```diff
@@ -58,3 +58,25 @@ keys:
                       name:
                         type: str
                         description: Dynamic Prefix List Name
+      control_functions:
+        type: dict
+        description: Configure routing control function to control which routes to filter and accept from a peer or routing protocol.
```

Comment:
you can also update / manipulate routes, so maybe change to "... which routes to filter, update and accept..."

Reply:
done.